ant-networking/src/cmd.rs

```diff
@@ -1212,13 +1233,13 @@ impl SwarmDriver {
 
         // get closest peers from buckets, sorted by increasing distance to the target
         let kbucket_key = target.as_kbucket_key();
-        let closest_k_peers: Vec<PeerId> = self
+        let closest_k_peers: Vec<(PeerId, Addresses)> = self
             .swarm
             .behaviour_mut()
             .kademlia
-            .get_closest_local_peers(&kbucket_key)
+            .find_closest_local_peers(&kbucket_key, &self.self_peer_id)
```

Comment:
We ignore `self_peer_id` here. This seems correct when trying to fetch candidates to send out replicate cmd to.

Reply:
better put this as a comment in code ?